Update go-to-market-groups.md

irenareedy opened the pull request:
Updated and added in event process

irenareedy committed:
Update go-to-market-groups.md

Updated and added in event process

Sampfluger88 reviewed (changes requested):
@irenareedy schedule some time so we can go over this together.

Sampfluger88 committed:
Update go-to-market-groups.md

Sampfluger88 reviewed (changes requested):
@irenareedy to make changes and schedule more time with Sam

irenareedy committed:
Merge branch 'main' into irenareedy-event-process

irenareedy committed:
Update go-to-market-groups.md

Headings updated

irenareedy reviewed (commented):
Headings have been chnaged

irenareedy committed:
Merge branch 'main' into irenareedy-event-process

irenareedy committed:
Merge branch 'main' into irenareedy-event-process

irenareedy committed:
Merge branch 'main' into irenareedy-event-process

irenareedy committed:
Merge branch 'main' into irenareedy-event-process

irenareedy committed:
Update go-to-market-groups.md

Deleted ""create-contact '  mention and changed it to Popl and Salesforce transfer

irenareedy committed:
Merge branch 'main' into irenareedy-event-process

irenareedy committed:
Merge branch 'main' into irenareedy-event-process

irenareedy committed:
Update go-to-market-groups.md

irenareedy committed:
Merge branch 'main' into irenareedy-event-process

lukeheath commented:
@Sampfluger88 Please review and merge ASAP as currently old PR.
